packages/bridge-ui/src/domain/transactions.ts

```diff
@@ -1,14 +1,62 @@
+import type { BigNumber, ethers } from "ethers";
+import type { Message, MessageStatus } from "./message";
+
+// export interface Data {
```

Comment:
commenting out block like this is a code smell imo

Reply:
yeah its not ready yet! sorry